packages/cyberstorm/src/components/componentStyles/Pagination.module.css

```diff
@@ -0,0 +1,20 @@
+.root {
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+  align-items: center;
+  justify-content: center;
+
+  width: 407px;
+  height: 36px;
+  padding: 0;
```

Comment:
I'm guessing this is here to eliminate the browser default padding for `<ul>`? This is also included in the `reset.css`, which carpet bombs everything to have `margin:0; padding:0;`. Of course we can't rely on us alwasys having that reset, and as such we should collectively have a chat about the feasibility of using `appearance:none` and `all:unset` on our components where applicable.

Reply:
You can leave `padding:0;` here for now as it's the desired result to reset such browser defaults for `<ul>` anyway. The ideas I mentioned shouldn't be made a blocker right now.